configuration/config-txt/codeclicence.md

```diff
@@ -1,6 +1,6 @@
 # Licence key and codec options in config.txt
 
-Hardware decoding of additional codecs on the Pi 3 and earlier models can be enabled by [purchasing a licence](http://swag.raspberrypi.org/collections/software) that is locked to the CPU serial number of your Raspberry Pi.
+Hardware decoding of additional codecs on the Pi 3 and earlier models can be enabled by [purchasing a licence](https://swag.raspberrypi.org/collections/software) that is locked to the CPU serial number of your Raspberry Pi.
```

Comment:
@JamesH65 It might be worth linking this directly to https://codecs.raspberrypi.org/ instead?